bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
@crungehottman For your second question, the simple unit test you linked looks like a reasonable start to me, though I have a few recommendations:

0. You can use [```with open```](https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects) (see the end of that section) to open the file, rather than having to remember to close the file. It's a small change, but it's a nice way to do file stuff in Python.
1. I actually didn't know about that click thing you linked me, but it looks pretty sweet. I'd use that. Another way you could do it is to ask Python for a [tempfile](https://docs.python.org/2.7/library/tempfile.html) and pass it to the click client. Either way is fine - I say try out that click thing since it sounds cool.
2. I'm not sure why a modified version of @cverna's test would cause his to fail - can you update this PR with that code so I can take a look? Maybe there's some shared object between them that just needs to be separate or something like that.

Oh, and the reason so many things have a test_url_flag is that there used to be no tests at all on the client and one day I wanted to add a ```--url``` flag to the client. So I had to write tests for my new code, and thus those were the first client tests. Now there are some more tests, but those are still there. Ideally, we'd have lots of tests in there - we're getting there!

Hopefully I got the questions you've mentioned so far, let me know if there's anything else you need to know!